app/views/_includes/summary-cards/qualifications.html

```diff
@@ -1,7 +1,28 @@
 {# Show this if the user says they have qualification to enter #}
 {% if data.anyQualifications == "Yes" %}
 
-  {{ xGovukSummaryCard({
+{% if data.anyQualifications == "Yes" %}
```

Comment:
We can simplify this page slightly, to only have this question appear once in the code. 

If we remove the if statement around this question (line 4 + line 23).